src/CardGalleryItem/CardGalleryItem.uni.driver.js

```diff
@@ -25,34 +25,86 @@ const getSecondaryAction = async base => {
 
 const cardGalleryItemDriverFactory = base => ({
   ...baseUniDriverFactory(base),
+  /**
+   * Gets the cards title.
+   * @returns {Promise<string | null>}
+   */
   getTitle: async () => {
     const title = getTitle(base);
     return (await title.exists()) ? title.text() : null;
   },
+  /**
+   * Gets the badge node element.
+   * @returns {Promise<any>}
+   */
   getBadge: async () => {
     const badge = getBadge(base);
     return (await badge.exists()) ? badge._prop('firstChild') : null;
   },
+  /**
+   * Gets the cards subtitle.
+   * @returns {Promise<string | null>}
+   */
   getSubtitle: async () => {
     const subtitle = getSubtitle(base);
     return (await subtitle.exists()) ? subtitle.text() : null;
   },
+  /**
+   * Gets the background image URL.
+   * @returns {Promise<string | null>}
+   */
   getBackgroundImageUrl: async () => await getOverlayDriver(base).getMediaUrl(),
+  /**
+   * Clicks on the primary action.
+   * @returns {Promise<void>}
+   */
   click: async () => (await getPrimaryAction(base)).click(),
+  /**
+   * Gets the primary action label.
+   * @returns {Promise<string>}
+   */
   getPrimaryActionLabel: async () => (await getPrimaryAction(base)).text(),
+  /**
+   * Return a boolean for the primary action clickability.
+   * @returns {Promise<boolean>}
+   */
   isPrimaryActionDisabled: async () =>
     (await getPrimaryAction(base))._prop('disabled'),
+  /**
+   * Clicks on the primary action button.
+   * @returns {Promise<void>}
+   */
   clickOnPrimaryAction: async () => (await getPrimaryAction(base)).click(),
+  /**
+   * Gets the secondary action label.
+   * @returns {Promise<string>}
+   */
   getSecondaryActionLabel: async () => (await getSecondaryAction(base)).text(),
+  /**
+   * Clicks on the secondary action.
+   * @returns {Promise<void>}
+   */
   clickOnSecondaryAction: async () => (await getSecondaryAction(base)).click(),
+  /**
+   * Gets the the settingsMenu node.
+   * @returns {Promise<node>}
+   */
   getSettingsMenu: async () => {
     await hover(base);
     const settingsMenu = getSettingsMenu(base);
     return (await settingsMenu.exists())
       ? settingsMenu._prop('firstChild')
       : null;
   },
+  /**
+   * Gets the background image element.
+   * @returns {Promise<any>}
+   */
   getBackgroundImageNode: async () => getOverlayDriver(base).getMediaNode(),
+  /**
+   * Trigger hover state on the element.
```

Comment:
`Triggers` is for events and callbacks

```suggestion
   * Hovers the component.
```